test/integration/controller/core/localqueue_controller_test.go

```diff
@@ -157,13 +157,19 @@ var _ = ginkgo.Describe("Queue controller", func() {
 		workloads := []*kueue.Workload{
 			testing.MakeWorkload("one", ns.Name).
 				Queue(queue.Name).
-				Request(corev1.ResourceCPU, "2").Obj(),
+				Request(resourceGPU, "2").
+				NodeSelector(map[string]string{resourceGPU.String(): flavorModelC}).
```

Comment:
Rather than complicating the workload... do we need the ResourceFlavors to be about GPUs or to have labels?
Can we just remove those instead?

Reply:
Sure. I use the label only for setting flavors to the assignment field in L204.
So, we can remove those labels by manually adding flavors to the field in L204.

Reply:
I'm not sure I understand.

Reiterating my question:
Do we need resourceFlavors in lines 42 and 43 to have labels?

Reply:
I use those labels only for automatically constructing Admission in L198-L204. So instead of using labels to construct Admissions automatically, we can define like https://github.com/kubernetes-sigs/kueue/blob/d84d5cfbb531672fbbc9297e9e6797fc83a33766/test/integration/controller/core/clusterqueue_controller_test.go#L182-L193.

This means we do not need those labels, and then I will remove those labels.

Reply:
@alculquicondor I updated this PR. Does it look good?

Reply:
yes, much better to read.